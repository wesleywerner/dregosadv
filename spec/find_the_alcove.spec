superbrief
gonear waterf portal
go north
examine alcove
go temple
examine glyphs
examine air terjun
consult journal about air terjun
go out
look behind waterfall
search waterfall
