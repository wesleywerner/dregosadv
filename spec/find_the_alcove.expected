  THIS GAME IS IN DEBUG MODE
  
  Dr Ego and the egg of Man-Toomba
  An Interactive Fiction Story
  by Special Agent.
  Release 2 / Serial number 400410 / Inform v6.33 Library v6.12.3pre SX
  
  The Maro River (in the canoe)
. 
  In the canoe I can also see a trap (which is closed and empty), a fishing
  spear and a fishing net.
  
> >  Dr Ego and the egg of Man-Toomba is now in its "superbrief" mode, which
  always gives short descriptions of locations (even if you haven't been
  there before).
  
> >  Waterfall
  A temple is due south. A rope bridge is due west.
  
> >  That way is the waterfall -- I don't need a shower right now.
  
> >  I can't see any such thing.
  
> >  Temple
. 
> >  The engravings cover all of the walls. My attention is drawn to a circular
  pattern in the glyphs as they converge and surround one large glyph -- I
  recognize this glyph as "air terjun".
  
> >  I may have some information about "air terjun" in my journal.
  
> >  Here is an underlined passage about air terjun -- That symbol indicates a
  hidden thing, more specifically, it means "behind the falling water".
  
> >  Waterfall
  A temple is due south. A rope bridge is due west.
  
> >  I lean into the narrow space behind the cascading water and see an alcove
  behind the waterfall.
  
  [I scored my first point! Type SCORE/FULL to list them.]
  
> >  I see the alcove behind the cascading water.
  
> >